libs/core/src/lib/card/card.component.scss

```diff
@@ -0,0 +1,890 @@
+@import '~fundamental-styles/dist/numeric-content';
+// @import '~fundamental-styles/dist/card';
+
+// TODO delete it once card styles is released
```

Comment:
this can be deleted now since we brought 0.12

Reply:
Done